src/Compilers/Core/Portable/Binding/BindingDiagnosticBag.cs

```diff
@@ -353,5 +353,20 @@ public override int GetHashCode()
         {
             return Diagnostics.GetHashCode();
         }
+
+        public bool HasAnyErrors() => Diagnostics.HasAnyErrors();
+
+        public bool HasAnyResolvedErrors()
+        {
```

Comment:
Would `Diagnostics.HasAnyResolvedErrors()` work?

Reply:
> Would ```Diagnostics.HasAnyResolvedErrors()``` work?

Not sure what API you have in mind. ```Diagnostics``` here is an ```ImmutableArray<Diagnostic>```.

Reply:
nit: consider factoring `(diagnostic as DiagnosticWithInfo)?.HasLazyInfo != true && diagnostic.Severity == DiagnosticSeverity.Error` into a `IsResolvedError` method that can be shared with existing `DiagnosticBag.HasAnyResolvedErrors()`.